.github/workflows/check-tests.yml

```diff
@@ -56,8 +56,18 @@ jobs:
             */*/node_modules
           key: ${{ runner.os }}-${{ hashFiles('**/package-lock.json') }}
 
-      - name: Test 🧪
+      - name: Prepare Tests
         working-directory: './packages/millicast-sdk'
-        run: |
-          npm install
-          npm test
+        run: npm ci
+
+      - name: Unit Testing 🧪
+        working-directory: './packages/millicast-sdk'
+        run: npm run test-unit
+
+      - name: End-2-End Testing
+        working-directory: './packages/millicast-sdk'
+        run: npm run test-e2e
+        env:
+          STREAM_NAME: ${{secrets.TEST_STREAM_NAME}}
```

Comment:
I think we should move TEST_STREAM_NAME and TEST_ACCOUNT_ID to action variables, so we can see the value in the log.  TEST_PUBLISH_TOKEN could be secret. 

And I already set up PUBLISHER_DEMO_ACC_ID(vars) and  PUBLISHER_DEMO_TOKEN(secrets) for actions, you could use it here if you don't need dedicate ones.

Reply:
Which is the PUBLISHER_DEMO_TOKEN's stream name and account id associated in the variables?